packages/ui/src/SegmentedControl.tsx

```diff
@@ -1,113 +1,60 @@
 import React from "react";
+import {Pressable, View} from "react-native";
 
-import {Box} from "./Box";
-import {
-  SegmentedControlProps,
-  SegmentedControlPropsMultiSelect,
-  SegmentedControlPropsSingleSelect,
-  SurfaceColor,
-} from "./Common";
+import {SegmentedControlProps} from "./Common";
 import {Text} from "./Text";
+import {useTheme} from "./Theme";
 
 export const SegmentedControl = ({
   items,
   onChange = () => {},
-  multiselect = false,
-  selectLimit = 1,
-  ...props
+  size = "md",
```

Comment:
So in the figma docs she has two sizes - large and default. Any reason to use "md" here instead of "default" ?

Reply:
I noted to talk Jo, it's kinda weird to have different sizes.